### PR TITLE
Remove Archetypes

### LINKED ISSUE #9: class security declaration warning
`Class plone.app.drafts.storage.Storage has a security declaration for nonexistent method 'drafts'`

### PR DESCRIPTION
**Pull request by petschki**
fixes #9

### CODE REVIEW

**Comment by jensens** [comment]
In this step I would go Python 3 only as well.

**Comment by petschki** [comment]
@jensens I've removed python 2.7 information and tests ... could not find any py2.7 compat code (six etc.) in modules ... am I missing something?

File: CHANGES.rst [hunk 1]
```diff
@@ -1,10 +1,12 @@
 Changelog
 =========
 
-1.1.4 (unreleased)
+2.0.0 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- remove Archetypes. This version targets Plone 5.2+ only.
```
Comment (jensens) [inline CHANGES.rst]:
```suggestion
- remove Archetypes. This version targets Plone 5.2+ with Python 3.6+ only.
```

**Review by jensens — APPROVED** [review]
one tiny addition to changelog? otherwise all fine, thx!

**Pull request merged**